File: app/src/main/java/com/scottmangiapane/cardclicks/game/Card.java
package com.scottmangiapane.cardclicks.game;

public class Card {
    public final int drawable;
    public final Color color;
    public final Number number;
    public final Shape shape;
    public final Type type;

    public Card(int drawable, Color color, Number number, Shape shape, Type type) {
        this.drawable = drawable;
        this.color = color;
        this.number = number;
        this.shape = shape;
        this.type = type;
    }
}

File: app/src/main/java/com/scottmangiapane/cardclicks/game/Card.kt
package com.scottmangiapane.cardclicks.game

@JvmRecord
data class Card(
    val drawable: Int,
    @JvmField val color: Color,
    @JvmField val number: Number,
    @JvmField val shape: Shape,
    @JvmField val type: Type
)
